src/platform/input/game-pads.js

```diff
@@ -135,110 +720,116 @@ class GamePads {
         }
 
         if (this.gamepadsSupported) {
-            const padDevices = navigator.getGamepads ? navigator.getGamepads() : navigator.webkitGetGamepads();
+            const padDevices = navigator.getGamepads() || navigator.webkitGetGamepads();
```

Comment:
This will not work either, since `navigator.getGamepads` will throw in the case that only `navigator.webkitGetGamepads` exists.